addons/l10n_uy/models/account_move.py

```diff
@@ -0,0 +1,104 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, _
+from odoo.exceptions import ValidationError
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    def _get_uruguayan_vat_taxes(self):
+        """ Found the VAT tax groups defined in Uruguay """
+        data_records = self.sudo().env['ir.model.data'].search([
+            ('module', '=', 'account'),
+            ('model', '=', 'account.tax.group'),
+            '|', '|',
+            ('name', 'ilike', 'tax_group_iva_10'),
+            ('name', 'ilike', 'tax_group_iva_22'),
+            ('name', 'ilike', 'tax_group_exenton'),
+        ])
+        group_taxes = self.env['account.tax.group'].search([
+            ('id', 'in', data_records.mapped('res_id')),
+            ('country_id', '=', self.env.ref('base.uy').id)])
+        return group_taxes
+
+    def _check_uruguayan_invoices(self):
+        """ Check that the UY invoices are vaid
+        * the partner of the invoice should have a valid identification type
+        * each invoice line should have one and only one vat tax
+        * export invoices should not have any vat tax.
```

Comment:
We can use VAT 0% (actually VAT Exempt)
From the user and functional point of view will be the same.

But for the technical side, If we want to use Exempt tax on Export Invoices then we need also to add a validation to ensure that they are using Exempt and not another VAT tax. And, If we want it to make it easier for the user maybe we could add some fiscal positions but not sure if we want to add more complexity. @ren-odoo @fvz-odoo @MarianaLuzTorres

More from the technical point of what we have in EDI, Exempt tax for regular invoices we decode them to send some info to DGI taking into account the tax type. For Expo invoices, we used a different logic because we needed to encode the values differently.

Reply:
@jco-odoo considering your comments we're plannig on adding the fiscal position approach we did for Peru https://github.com/odoo/odoo/blob/16.0/addons/l10n_pe/data/fiscal_position_data.xml, does it make sense for  you? 

@zaoral @fvz-odoo

Reply:
Hi @jco-odoo @ren-odoo 
I have added the fiscal positions as in PE.
Also, I removed the check for exportation invoices

Please could you review @ren-odoo 
Best Regards